AUTHORS

```diff
@@ -1,228 +1 @@
-Aaron Meihm
-Akash Mathur
-Albiona
-Alex
-Alex Davis
-Allen Short
-amantayal6797
-Andreas Textor
-Andrew Chilton
-Ankita Shrivastava
-Anna Sobiepanek
-Anny
-Anwesh Nayak
-apoorvasingh17
-Arshad Kazmi
-Ashwin Patil
-Austin King
-ava14
-Barry Chen
-Beatriz Rizental
-Ben Bangert
-Benson Wong
-besnik@programeshqip.org
-Bianca Danforth
-bini11
-Brandon Ebersohl
-Brian Warner
-Bryan Olsson
-byrw
-championshuttler
-Chiamaka-Uzuegbu
-Chris Heilmann
-Chris Karlof
-Christian Murphy
-ckarlof
-Claas Augner
-claubatista
-Cronus1007
-César Carruitero
-Dan Callahan
-Dan Schomburg
-Danny Amey
-Danny Coates
-Dave Justice
-dave justice
-Deeksha Tiwari
-Deepti
-dependabot[bot]
-DEV Akash Mathur
-dhruva248
-Divya Biyani
-divyabiyani
-dschom
-Dustin Lactin
-Edouard Oger
-Edwin Wong
-Emin Mastizada
-Eric Le Lay
-Erika Jonell
-Farhan Khan
-feelcoder
-Florian Dieminger
-Fmstrat
-Francesco Lodolo
-Francesco Lodolo [:flod]
-Francois Marier
-fxa-bananafox[bot]
-Gaurangi
-Glen Mailer
-Greg Guthe
-GuptaTanisha
-Gurjeet Singh
-Hannah Quay-de la Vallee
-hannahqd
-Heather Booker
-Hector Zhao
-hritvi
-Hritvi Bhandari
-Ian Bicking
-irrationalagent
-Ishmam Zaman
-Ivo Plamenac
-Jackie Munroe
-jackyzy823
-Jamon Camisso
-Jane Kotovich
-Jared Hirsch
-Jarek
-Jason Strutz
-jbonacci
-Jed Parsons
-Jody Heavener
-Johannes Faust
-John Gruen
-John Morrison
-johngruen
-Jon Buckley
-Jon Petto
-Josef Weldemariam
-jotes
-JR Conlin
-Juan García Basilio
-Julian Poyourow
-Jurgen Brunink
-ka7
-Karan Sapolia
-Karina Bekbayeva
-karinabk
-kik-kik
-Kit Cambridge
-Kohei Yoshino
-Kurt Bauer
-Larissa Gaulia
-larissagaulia
 Lauren Zugai
```

Comment:
Sus

Reply:
I reverted in a separate commit instead of force-pushing.